app/views/summary/UKSitesSummary.scala

```diff
@@ -91,28 +91,30 @@ object UKSitesSummary {
       )
   }
 
-  def getHeadingAndSummary(userAnswers: UserAnswers, isCheckAnswers: Boolean, subscription: RetrievedSubscription)
```

Comment:
Is this function not needed elsewhere?

Reply:
discussed and resolved.  Base method added for use in other journeys